tests/multitrack.rs

```diff
@@ -1,48 +1,53 @@
 mod paths;
 
+#[path = "common/mod.rs"]
+mod common;
+
 #[test]
 fn multitrack_video_with_subtitles_and_sound() {
     let mp4_with_subtitles_and_sound = std::path::Path::new(paths::SAMPLE_BASE_PATH)
         .join("rerun404_avc_with_subtitles_and_sound.mp4");
 
-    let video = re_mp4::Mp4::read_file(mp4_with_subtitles_and_sound).unwrap();
-
+    let (video, data) = re_mp4::Mp4::read_file(mp4_with_subtitles_and_sound).unwrap();
     assert_eq!(video.tracks().len(), 3);
     assert_eq!(video.moov.mvhd.next_track_id, 4);
 
     // Video track.
     {
         let track = video.tracks().get(&1).unwrap();
+        let data = common::get_sample_data(&data, track);
         assert_eq!(track.kind, Some(re_mp4::TrackKind::Video));
         assert_eq!(track.codec_string(&video), Some("avc1.640028".to_owned()));
         assert_eq!(track.track_id, 1);
         assert_eq!(track.width, 600);
         assert_eq!(track.height, 600);
         assert!(!track.samples.is_empty());
-        assert!(!track.data.is_empty());
+        assert!(!data.is_empty());
```

Comment:
nit: these are equivalent to checking for track having any samples with non-zero range, so don't need to extract sample_data for real. Doesn't matter much though, mostly a bit of a conceptual _meh_ in the test

Reply:
:shrug: just trying to preserve the original test, seems fine to leave it as is here